easycv/datasets/classification/pipelines/auto_augment.py

```diff
@@ -182,6 +191,41 @@ def merge_hparams(policy: dict, hparams: dict):
     return policy
 
 
+@PIPELINES.register_module()
+class ThreeAugment(object):
```

Comment:
use MMAutoAugment, refer to: `easycv/datasets/classification/pipelines/auto_augment.py` MMAutoAugment

Reply:
Three augment is a special and optimized augment for vit, AutoAugment may not  provide same implementation

Reply:
done
merge ThreeAugment into MMAutoAugment